PMurphy1940/ContractThis
# Bids and chat

# Description
A user can send a work bid request to a subcontractor. A subcontractor can see the work request and choose to accept it. The user is shown accepted an outstanding bids on the component detail view. If there is no bid on the current component, the user sees the 'Search Subcontractors' affordance.
## Type of change
# Testing Instructions
On the components route => select a component => choose a contractor from the search function => click the chat button(which opens the bid request form) => submit the bid. The component card will then show an outstanding bid at the bottom. Log in as the selected subcontractor and you will see the work request. Clicking the Accept button will add the current date to the 'SubAccepted' field in the Bid database. Again log in as the primary user, and you will see that bid now reads as an accepted bid and the labor (fee) is added to the total expenditures.
# Checklist:
